libs/api/domains/rsk-company-info/src/lib/graphql/models/rskCompany.model.ts

```diff
@@ -0,0 +1,45 @@
+import { Field, ObjectType, ID } from '@nestjs/graphql'
+import { RskCompanyAddress } from './rskCompanyAddress.model'
+import { RskCompanyFormOfOperation } from './rskCompanyFormOfOperation.model'
+import { RskCompanyLink } from './rskCompanyLink.model'
+import { RskCompanyRelatedParty } from './rskCompanyRelatedParty.model'
+import { RskCompanyVat } from './rskCompanyVat.model'
+
+@ObjectType()
+export class RskCompanyInfo {
+  @Field(() => [RskCompanyFormOfOperation], { nullable: true })
+  formOfOperation?: RskCompanyFormOfOperation[]
+
+  @Field(() => [RskCompanyAddress], { nullable: true })
+  address?: RskCompanyAddress[]
+
+  @Field(() => [RskCompanyRelatedParty], { nullable: true })
+  relatedParty?: RskCompanyRelatedParty[]
+
+  @Field(() => [RskCompanyVat], { nullable: true })
+  vat?: RskCompanyVat[]
```

Comment:
I generally prefer defaulting these kind of fields to empty arrays instead of nullable, unless there's a reason to differentiate the two.